ls1intum/Artemis
# [Enhancement] Change Quiz Layout of Drag Items

### Checklist

### Motivation and Context



The Drag Items in Quiz Drag&Drop Questions are spaced wrongly.

### Description


- Fix spacing of items
- Fix overflow for long text items

### Steps for Testing


1. Log in to Artemis
2. Navigate to a quiz exercise with d&d questions.
3. Check the layout for various screen sizes and number of items (text & image items).

### Screenshots



<img width="301" alt="Bildschirmfoto 2020-07-21 um 17 43 36" src="https://user-images.githubusercontent.com/13920539/88075972-cda75b80-cb79-11ea-8a1a-268269fd6619.png">
<img width="731" alt="Bildschirmfoto 2020-07-21 um 17 43 52" src="https://user-images.githubusercontent.com/13920539/88075988-d13ae280-cb79-11ea-9479-9d25fa1448ae.png">
